src/components/panes/Quality/QualityModal/QualityModal.tsx

```diff
@@ -23,6 +23,7 @@ const useStyles = makeStyles((theme) =>
   createStyles({
     modal: {
       width: '100%',
+      maxWidth: '600px',
     },
     content: {
       padding: '3em',
```

Comment:
Let's make the padding the same as the Connectivity Modal. I think the 2em padding is nice for mobile: 
```
 padding: '3em',
      [theme.breakpoints.down('sm')]: {
        padding: '2em',
      },
 ```